postcss.config.js

```diff
@@ -1,7 +1,20 @@
+const autoprefixer = require('autoprefixer');
+const assets = require('postcss-assets');
+
+const CF_DIST = process.env.CLOUDFRONT_DIST;
```

Comment:
I'm not sure this would work since this retrieves the environment variable at compile time but we may change it without rebuilding the static assets. I believe heroku will just restart Django if we set a new environment variable. @blarghmatey @shaidar can you confirm?